vars/uploadToCixLogs.groovy

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/groovy
+
+def call(def stage) {
+  def remoteDir = '/home/www/html/'.plus("${env.BUILD_TAG},${stage}".replaceAll('/', ','))
```

Comment:
I would define the remoteDir in cix config instead of here, so you don't have to release a new version of the lib if the path changes

Reply:
Good catch